simulator/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM rust:1.70.0-bullseye
+
+RUN apt update && apt install -y cmake clang build-essential llvm
+RUN rustup target add wasm32-unknown-unknown
+
+ADD simulator /app/simulator
+ADD radix-engine /app/radix-engine
+ADD radix-engine-interface /app/radix-engine-interface
+ADD sbor /app/sbor
+ADD sbor-derive /app/sbor-derive
+ADD sbor-derive-common /app/sbor-derive-common
+ADD utils app/utils
+ADD scrypto-schema /app/scrypto-schema
+ADD radix-engine-common /app/radix-engine-common
+ADD radix-engine-derive /app/radix-engine-derive
+ADD native-sdk /app/native-sdk
+ADD radix-engine-macros /app/radix-engine-macros
+ADD radix-engine-profiling /app/radix-engine-profiling
+ADD radix-engine-store-interface /app/radix-engine-store-interface
+ADD radix-engine-stores /app/radix-engine-stores
+ADD transaction /app/transaction
+ADD radix-engine-queries /app/radix-engine-queries
+ADD assets /app/assets
```

Comment:
Can we instead (or as well as) use a docker ignore file? For us developers, lots of these paths will have massive target folders (like, 200GB of target folders :D) which will need ignoring.

Reply:
added the dockerignore file with the target/ folder. please do let me know if there's anything else that should go in there